app/presenters/sort_option_presenter.rb

```diff
@@ -0,0 +1,36 @@
+class SortOptionPresenter
```

Comment:
Could we maybe find a different name for this? - Or maybe move it into ```SortPresenter``` somehow. The difference between the name ```SortOptionPresenter``` and ```SortPresenter``` is not immediately obvious from the name alone

Reply:
I’d prefer to keep the classes in separate files. Maybe we could rename sort presenter to something like sort collection presenter?